### ISSUE TITLE
Question on netcdf4 required by vat

### ISSUE THREAD

**Issue by hmorzaria** [cddesja/vat#3]
Christopher:

Hi, I've tried to install vat but run into an error with dependency on netcdf4. I wanted to see if you had any suggestions. 
If I am running R in a 64-bit Windows 7 system. If I run 32-bit R,install netcdf4 for 32bit and try to install vat I get an error that netcdf is not installed on arch x64. Conversely if I run 64-bit R,install netcdf4 for 64bit and try to install vat I get an error that netcdf is not installed on arch xi386.

Thanks!

Hem Nalini Morzaria

> install_github(repo = "cddesja/vat", )
Downloading github repo cddesja/vat@master
Installing vat
"C:/Users/HemNalini.MorzariaLu/Documents/R/R-3.1.2/bin/i386/R" --vanilla CMD INSTALL  \
  "C:/Users/HemNalini.MorzariaLu/AppData/Local/Temp/1/RtmpMPyZWR/devtools397c5f011eab/cddesja-vat-5ee7bff"  \
  --library="C:/Users/HemNalini.MorzariaLu/Documents/R/win-library/3.1" --install-tests 

* installing *source* package 'vat' ...
** R
** preparing package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
*** arch - i386
*** arch - x64
Error : package 'ncdf4' is not installed for 'arch = x64'
Error: loading failed
Execution halted
ERROR: loading failed for 'x64'
* removing 'C:/Users/HemNalini.MorzariaLu/Documents/R/win-library/3.1/vat'
Error: Command failed (1)

**Comment by cddesja** [comment]
Have you installed ncdf4 prior to loading the package? On Windows, it looks like you don't have a binary available.  Can you try, launching R in 64 bit, and
```{r}
install.packages("ncdf4", type = "source')
```
If so, try
```{r}
library("ncdf4")
```
Does that load OK? If so, then run
```{r} 
devtools::install_github(repo = "cddesja/vat")
```
I haven't tested `vat` on a Windows machine, so I'd be happy to hear about any success/failure you might encounter. Also, it might depend on where exactly `R` is installed. I can get access to a Windows machine, however, if this bug persists.

**Comment by cddesja** [comment]
I just installed a Windows virtual machine and was unable to replicate your issue.  However, I did a few things that might be different from what you've done.

1) I installed R-64 only to C:/
2) I installed http://www.unidata.ucar.edu/downloads/netcdf/ftp/netCDF4.3.3.1-NC4-64.exe to C:/
3) I installed the binary of the ncdf4 package from here: http://cirrus.ucsd.edu/~pierce/ncdf/
4) I then ran `install.packages("ncd4_1.12.zip", repo = NULL, type = "source")`
5) Then I install `vat` follow the instructions on the GitHub site.

See if that makes a difference.
